geopandas/geodataframe.py

```diff
@@ -1696,9 +1696,11 @@ def dissolve(
 
         Parameters
         ----------
-        by : string, default None
-            Column whose values define groups to be dissolved. If None,
-            whole GeoDataFrame is considered a single group.
+        by : str or list-like, default None
+            Column(s) whose values define the groups to be dissolved. If None,
+            the entire GeoDataFrame is considered as a single group. If a list-like
+            object is provided, the values in the list are treated as categorical labels,
+            and polygons will be combined based on the equality of these categorical labels.
```

Comment:
```suggestion
            object is provided, the values in the list are treated as categorical
            labels, and polygons will be combined based on the equality of
            these categorical labels.
```

All lines need to be under 88 chars long